From: Vlastimil Babka <vbabka@suse.cz>
Subject: temporarily disable debug_pagealloc
Patch-Mainline: Never, temporary workaround
References: bsc#1159096

Enabling debug_pagealloc=on prevents proper booting due to jump label init
order. Until it's fixed, disable this non-essential debugging feature.

Signed-off-by: Vlastimil Babka <vbabka@suse.cz>

---
 mm/page_alloc.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -639,6 +639,9 @@ static int __init early_debug_pagealloc(
 {
 	bool enable = false;
 
+	/* Temporary workaround, bsc#1159096 */
+	return 0;
+
 	if (kstrtobool(buf, &enable))
 		return -EINVAL;
 
